src/App.js

```diff
@@ -1,8 +1,26 @@
 import React from 'react';
 import './App.scss';
+import PropTypes from 'prop-types';
+import Sum from './components/Sum';
 
 const App = () => (
-  <h1>React sum</h1>
+  <div className="app">
+    <h1>React sum</h1>
+    <Text a={2} b={2} />
+    <Text a={10} b={0} />
+    <Text a={-5} b={5} />
+  </div>
 );
 
+const Text = ({ a, b }) => (
+  <p className="text">
+    {`Sum of ${a} and ${b} is ${Sum(a, b)}`}
+  </p>
+);
```

Comment:
You need to have `Sum` component that renders certain text.
Really don't need to create separate function to add two numbers.